doc/ReleaseNotes.md

```diff
@@ -1,6 +1,11 @@
 # SVG.NET Release Notes
 The release versions are NuGet releases.
 
+## [Version X.X.X](https://www.nuget.org/packages/Svg/X.X.X)  (20XX-XX-XX)
```

Comment:
Thanks - we usually just use `## Unreleased` here to avoid dead links.

Reply:
Updated